source/class/qx/tool/cli/Cli.js

```diff
@@ -341,6 +341,9 @@ Version: v${await qx.tool.config.Utils.getQxVersion()}
       await compilerApi.load();
       let config = compilerApi.getConfiguration();
 
+      // Validate compile.json against the schema
```

Comment:
The title of this PR says it now also validates compile.js. Should this comment mention that too?